query/es/es_domain_encryption_between_nodes.sql

```diff
@@ -0,0 +1,18 @@
+select
```

Comment:
@sayan133 Can you please rename this query file to `es_domain_node_to_node_encryption_enabled.sql`?